better-font-awesome.php

```diff
@@ -21,6 +21,11 @@
  * GitHub Plugin URI: https://github.com/MickeyKay/better-font-awesome
  */
 
+// Exit if accessed directly
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
 add_action( 'plugins_loaded', 'bfa_start', 5 );
 /**
```

Comment:
Try to keep inline code and class defines in seperate files. Makes for more tidy code and better flexibility. ```Better_Font_Awesome_Plugin``` class should be in it's own file by its self.